content/docs/get-started/_index.md

```diff
@@ -15,31 +15,36 @@ aliases:
 
 <div class="flex max-w-2xl">
     <p class="text-xl leading-relaxed text-gray-600">
-        Step-by-step guides for creating, deploying, and managing infrastructure with
-        Pulumi on the cloud using your favorite language.
+        Pulumi is a modern infrastructure-as-code platform that allows you to use common programming languages, tools, and frameworks, to provision, update, and manage cloud infrastructure resources.
+    </p>
+</div>
+
+<div class="flex max-w-2xl">
+    <p class="text-xl leading-relaxed text-gray-600">
+        Select one of the following options to quickly learn how to use Pulumi to spin up resources in the cloud within minutes.
     </p>
 </div>
 
 ## Choose Your Cloud
 
 <div class="tiles flex-wrap mt-4">
     <div class="pb-4 md:pr-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "aws" >}}">
+        <a class="tile p-8" href="{{< relref "/docs/get-started/aws/begin" >}}">
```

Comment:
I am assuming you are intentionally linking to `/begin` here? If so, let's remember to track this change in some metrics.

Reply:
Yup, that's correct. Part of this PR is to get people into the tutorial sooner by removing the intro page as part of the flow and pushing the relevant/useful info from that page into the main Getting Started overview page.